cornice/tests/validationapp.py

```diff
@@ -62,11 +62,8 @@ def get2(request):
 service3 = Service(name="service3", path="/service3")
 
 
-def _accept(request):
-    return ('application/json', 'text/json')
-
-
-@service3.get(accept=_accept)
+@service3.get(accept=lambda request: ('application/json', 'text/json'))
+@service3.put(accept=lambda request: 'text/json')
```

Comment:
- Having the services named ``serviceX`` offended my eyes too when having the first look the other day, so i totally second that.
- However, the ``service3`` is perfectly for testing this case of having a callable as argument to ``accept`` and can well carry both flavors (returning list vs. returning scalar) when renamed properly.

As said, i'm happy to add another PR after merging #350 and #351 to improve the shape of the test cases, especially regarding minor refactoring and proper renaming of the ``serviceX`` jungle.